worlds/yoshisisland/Regions.py

```diff
@@ -0,0 +1,225 @@
+        create_region(multiworld, player, locations_per_region, location_cache, "5-5"),
+        create_region(multiworld, player, locations_per_region, location_cache, "5-6"),
+        create_region(multiworld, player, locations_per_region, location_cache, "5-7"),
+        create_region(multiworld, player, locations_per_region, location_cache, "5-8"),
+        create_region(multiworld, player, locations_per_region, location_cache, "Raphael The Raven's Boss Room"),
+
+        create_region(multiworld, player, locations_per_region, location_cache, "6-1"),
+        create_region(multiworld, player, locations_per_region, location_cache, "6-2"),
+        create_region(multiworld, player, locations_per_region, location_cache, "6-3"),
+        create_region(multiworld, player, locations_per_region, location_cache, "6-4"),
+        create_region(multiworld, player, locations_per_region, location_cache, "Tap-Tap The Red Nose's Boss Room"),
+        create_region(multiworld, player, locations_per_region, location_cache, "6-5"),
+        create_region(multiworld, player, locations_per_region, location_cache, "6-6"),
+        create_region(multiworld, player, locations_per_region, location_cache, "6-7"),
+        create_region(multiworld, player, locations_per_region, location_cache, "6-8"),
+        create_region(multiworld, player, locations_per_region, location_cache, "Bowser's Room"),
+
+    ]
+    if world.options.extras_enabled.value == 1:
+        regions.insert(68, create_region(multiworld, player, locations_per_region, location_cache, "6-Extra"))
+        regions.insert(58, create_region(multiworld, player, locations_per_region, location_cache, "5-Extra"))
+        regions.insert(48, create_region(multiworld, player, locations_per_region, location_cache, "4-Extra"))
+        regions.insert(38, create_region(multiworld, player, locations_per_region, location_cache, "3-Extra"))
+        regions.insert(28, create_region(multiworld, player, locations_per_region, location_cache, "2-Extra"))
+        regions.insert(18, create_region(multiworld, player, locations_per_region, location_cache, "1-Extra"))
+
+    if world.options.minigame_checks.value >= 2:
+        regions.insert(74, create_region(multiworld, player, locations_per_region, location_cache, "6-Bonus"))
+        regions.insert(63, create_region(multiworld, player, locations_per_region, location_cache, "5-Bonus"))
+        regions.insert(52, create_region(multiworld, player, locations_per_region, location_cache, "4-Bonus"))
+        regions.insert(41, create_region(multiworld, player, locations_per_region, location_cache, "3-Bonus"))
+        regions.insert(29, create_region(multiworld, player, locations_per_region, location_cache, "2-Bonus"))
+        regions.insert(19, create_region(multiworld, player, locations_per_region, location_cache, "1-Bonus"))
+    multiworld.regions += regions
+
+    connect_starting_region(multiworld, player, world)
+

```

Comment:
```suggestion
    connect_starting_region(world)
        multiworld = world.multiworld
        player = world.player

```

Reply:
Is multiworld and player supposed to be indented, here? Errors out for me.

Reply:
It seems to work if I exclude those lines (but shorten the argument to just world), so I suppose I'll resolve this

Reply:
This one was a mistake, the def is the only one that matters.